cli.js

```diff
@@ -292,3 +295,49 @@ function exportAllCli(cli) {
       .then(() => exportData(tableName, null, region))
   }, { concurrency: 1 });
 }
+
+function wipeDataCli(cli) {
+  const tableName = cli.flags.table;
+
+  if (!tableName) {
+    console.error('--table is requred')
+    cli.showHelp();
+  }
+
+  return wipeData(tableName, cli.flags.file, cli.flags.region);
+}
+
+function wipeData(tableName, file, region) {
+  const dynamoDb = new AWS.DynamoDB({ region });
+
+  let n = 0;
+
+  const params = { TableName: tableName };
+  const scanPage = () => {
+    return bluebird.resolve(dynamoDb.scan(params).promise())
+      .then(data => {
+        const promises = [];
+        data.Items.forEach(item => {
+          const key = {
+            TableName: tableName,
+            Key: item
+          };
+          promises.push(dynamoDb.deleteItem(key).promise());
```

Comment:
This will execute all deletes in parallell. have you tested this on a big table? I guess the only reason why it works is because the pagination limits the amount of concurrent calls (how many?) If so, can you add a comment about this. A safer option might be to instead use bluebird.map(data.Items, item => {...}, { concurrency }) where concurrency is a constant saying how many requests to do in parallell.